Makefile

```diff
@@ -1,12 +1,8 @@
 ORG     ?= $(shell basename $(realpath ..))
 PKGS    := $(shell go list ./... | grep -v /vendor/)
 
```

Comment:
If you add 
```
export GO111MODULE := on
```
This will enable go modules independently of it's checked-out path. See the [relevant entry in the go modules wiki](https://github.com/golang/go/wiki/Modules#when-do-i-get-old-behavior-vs-new-module-based-behavior).

Reply:
Ah, that sounds good, so that I don't need the instruction like https://github.com/roboll/helmfile/pull/564#discussion_r278399438 for everyone?